kiss

```diff
@@ -1163,7 +1174,8 @@ pkg_updates() {
                 # ownership of files and directories in the rare
                 # case that the repository is owned by a 3rd user.
                 (
-                    user=$(stat -c %U "$PWD") || user=root
+                    user=$(ls -ld "$PWD") || user=root
+                    user=${user#* * } user=${user%% *}
```

Comment:
I've implemented the user calls here: https://github.com/kisslinux/kiss/commit/278c222902a38046012fdfc94e40269bd3378a9b#diff-4188679c1d8a284ccc41a6b601869e05 :+1:

Reply:
I've rebased on that commit